Makefile

```diff
@@ -26,6 +26,9 @@ endif
 run:
 	python3 -m luigi $(SCHEDULER) --module tasks.$(MOD_NAME) tasks.$(TASK)
 
+docker-run:
+	docker-compose run -d -e LOGGING_FILE=etl_$(MOD_NAME).log bigmetadata luigi --module tasks.$(MOD_NAME) tasks.$(TASK)
```

Comment:
What if we add for the LOGGING_FILE `etl_$(MOD_NAME)_$(TASK)`?

Reply:
The task also includes the parameters in the form of `--parameter value` that's why I'm using the module name

Reply:
Ahh is true! Cool!